portfolio/src/main/webapp/index.html

```diff
@@ -38,17 +37,17 @@ <h2 class="center">About</h2>
           Currently I am a STEP intern at Google, and I am excited to learn more about Javascript and web dev in general. 
           Other areas of interest are machine learning and computer vision. 
       </p>
-      <p >In my free time I like to relax by playing acoustic guitar and playing Overcooked! and CS:GO with my friends. 
+       <p >In my free time I like to relax by playing acoustic guitar and playing Overcooked! and CS:GO with my friends. 
           In pre-Corona times I also liked trying new restaurants in my city with friends.
           A fun fact about myself is that I can play five instruments: piano, guitar, tenor saxophone, ukulele, and harmonica.
            But I can't play the last 3 very well. Other things I'm interested in are
-          trivia (I played on my high school's team and I'd love to be on Jeopardy! someday), makeup, and cooking.  
-      </p>
-      <div class="blogpost">
+          trivia (I played on my high school's team and I'd love to be on Jeopardy! someday), makeup, and <a href="/cook.html">cooking</a>.  
+        </p>
+        <div class="blogpost">
         <p>I don't have any pets of my own, though I do really like bunnies. 
             Here's a photo of a cute rabbit that you can pet (by clicking):</p>
          <img id="bunny" onclick="bunnymsg()" style="height: 50px;width:80px;" src="img/bunny.jpg" alt="white rabbit"/>
-        <div id="bun_msg" style=" color:darkslategrey;"></div>
+         <div id="bun_msg" style=" color:darkslategrey;"></div>
```

Comment:
Your indentation looks a bit weird on this line and some of the lines above it. Try to maintain a 2 space independent between each container. e.g.
```
<div1>
  <div2>
    <div3> ....
```

Reply:
Yes, I missed this section when I went to fix my indents thanks for pointing out.